### ISSUE TITLE
duplicate neighbors

### ISSUE THREAD

**Issue by paulleo** [proxpero/Geodesy#1]
I'm getting the following neighbors:
["gcpvhgm", "gcpvhgm", "gcpvhgj", "gcpvhfv", "gcpvhfv", "gcpvhfv", "gcpvhgj", "gcpvhgm"]

Sure there shouldn't be any duplicates right?

**Comment by proxpero** [comment]
No, you're right. There should _not_ be duplicates. What was your original hash or location?

**Comment by olegbragin** [comment]
I also see the duplicates. 
Original location:   
- latitude : 56.08028609366816
- longitude : 9.239284694194794

Selected precision: .twentyFourHundredMeters
